modules/con-enabling-config-management-plugins-argo-cd-cr.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="enabling-config-management-plugins-argocd_{context}"]
+= Enabling Config Management Plugins in an Argo CD CR
+
+Argo CD provides support for Helm, Jsonnet, and Kustomize as built-in config management tools. To use a different config management tool, or to enable features not provided by the built-in config management tools, you can use the Config Management Plugin (CMP).
+
+In Argo CD, the CMP is specified as a sidecar container for the Argo CD repo server container. For more information, see "Config Management Plugins".
+
+In the {gitops-title} Operator, you can configure the Config Management plugin as a sidecar container in the Argo CD custom resource (CR). When you configure the sidecar container, you either specify an off-the-shelf or a custom-built container image. If you do not specify an image, the system uses the same image as the repo server for the plugin.
+
+To configure a sidecar container in the {gitops-title} Operator, add the `.spec.repo.sidecarContainers` key in the Argo CD CR. 
+
+.Example Config Management Plugin configuration
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+ name: my-argocd # <1>
+spec:
+ repo:
+   sidecarContainers:
+     - name: my-cmp # <2>
+       command: [/var/run/argocd/argocd-cmp-server]
+       image: my-image
+       securityContext:
+         runAsNonRoot: true
+         runAsUser: 999
+       volumeMounts: # <3>
+         - mountPath: /var/run/argocd
+           name: var-files
+         - mountPath: /home/argocd/cmp-server/plugins
+           name: plugins
+         - mountPath: /tmp
+           name: tmp
+         - mountPath: /home/argocd/cmp-server/config/plugin.yaml
+           subPath: plugin.yaml
+           name: cmp-plugin
+----
+<1> Specify the name of an Argo CD CR instance.
```

Comment:
Given that this is a concept and not a procedure, I _think_ these should be changed to `Specifies`

Reply:
Fixed.